src/app/visual/select-field/select-field-visual.component.html

```diff
@@ -1,48 +1,138 @@
-<br/>
-<div id="screenshot-select-field-single-mode-wrapper" style="padding: 5px 0;">
-  <div id="screenshot-select-field-single-mode">
+<div
```

Comment:
The vast majority of changes to this file are primarily code style. I also added a reactive form below to make manual testing easier :) .